### PR TITLE
json object has a default limit of 100kb

### PR DESCRIPTION
**Pull request by tobslob**

### CODE REVIEW

File: src/middleware.ts [hunk 1]
```diff
@@ -46,7 +46,7 @@ export function build(
   conf: SiberConfig = {}
 ) {
   // default middleware
-  app.use(express.json());
+  app.use(express.json({ limit: conf.requestBodyLimit }));
   app.use(express.urlencoded({ extended: false }));
```
Comment (noxecane) [inline src/middleware.ts]: `requestBodyLimit` should also affect `URLEncoded` requests. The name doesn't distinguish the type of request.

Comment (tobslob) [reply src/middleware.ts]: Alright. could this be a better name `requestPostLimit`?

Comment (noxecane) [reply src/middleware.ts]: I'm not talking about a name change. I'm saying the name implies it affects the body regardless of the type so it means both `JSON` and `URLEncoded` body types should be affected

**Comment by noxecane** [comment]
Rejecting this as your commit messages are nowhere near what is being fixed. And you didn't need to change the property name. Here is a better commit message:
- fix: `requestBodyLimit` has no influence on the body limits

Since you've already added `requestBodyLimit` in a previous PR, the only thing wrong with it is that it doesn't do anything, hence this commit message

**Pull request closed without merging**